clients/flash/air-client/src/org/bigbluebutton/air/common/views/skins/MobileToggleSwitchSkin.as

```diff
@@ -0,0 +1,382 @@
+package org.bigbluebutton.air.common.views.skins {

```

Comment:
It contains some parts of `spark.skins.ios7.ToggleSwitchSkin` under ASF v2.0 licence, with some customisation. What do you suggest? Adding Apache Flex Licence to the header?